enterprise_access/apps/bffs/serializers.py

```diff
@@ -28,3 +30,196 @@ class ErrorSerializer(BaseBFFMessageSerializer):
 
 class WarningSerializer(BaseBFFMessageSerializer):
     pass
+
+
+class BaseResponseSerializer(serializers.Serializer):
+    """
+    Serializer for base response.
+    """
+
+    errors = ErrorSerializer(many=True, required=False, default=list)
+    warnings = WarningSerializer(many=True, required=False, default=list)
+
+    def create(self, validated_data):
+        return validated_data
+
+    def update(self, instance, validated_data):
+        return validated_data
+
+    def to_representation(self, instance):
+        """
+        Override to_representation method to return ordered representation
+        with errors/warnings at the end of the response.
+        """
+        representation = super().to_representation(instance)
+
+        ordered_representation = OrderedDict(representation)
+
+        # Remove errors and warnings from the main response (they will be re-added at the end)
```

Comment:
nit: maybe just a slight revision to the comment to add "at the end **of the response**" and/or perhaps combine the lines 63-64 below all under a single comment block.

```python
# Temporarily remove errors and warnings from the main response, and re-add them show they appear at the end of the response.
errors = ordered_representation.pop('errors', [])
warnings = ordered_representation.pop('warnings', [])
ordered_representation['errors'] = errors
ordered_representation['warnings'] = warnings
```